### DISCUSSION TITLE
Not able to launch bot on Debian based Linux (Main System Chrome)

### DISCUSSION THREAD

**Discussion by Eribaeri** [jef/streetmerchant#1155] [Help]
### Streetmerchant not starting

installed Git
installed Node
updated Node to 14.15.01 (nvm)
installed Streetmerchant

I can't see any .env file, edited it on my PC, tried to get it to Linux, didn't work.
eribaeri@penguin:~/streetmerchant$ npm run start
it says:

ＳＴＲＥＥＴＭＥＲＣＨＡＮＴ
3.1.0

[12:21:16 PM] warn :: nvidia is deprecated in favor of bestbuy
[12:21:17 PM] info :: ℹ selected stores: nvidia
[12:21:17 PM] info :: ℹ selected series: 3060ti, 3070, 3080, 3090, rx6800, rx6800xt, rx6900xt, ryzen5600, ryzen5800, ryzen5900, ryzen5950, sonyps5c, sonyps5de, xboxss, xboxsx
[12:21:25 PM] error :: ✖ something bad happened, resetting streetmerchant in 5 seconds Failed to launch the browser process!
/home/eribaeri/streetmerchant/node_modules/puppeteer/.local-chromium/linux-818858/chrome-linux/chrome: error while loading shared libraries: libnss3.so: cannot open shared object file: No such file or directory

TROUBLESHOOTING: https://github.com/puppeteer/puppeteer/blob/main/docs/troubleshooting.md
 {
  "stack": "Error: Failed to launch the browser process!\n/home/eribaeri/streetmerchant/node_modules/puppeteer/.local-chromium/linux-818858/chrome-linux/chrome: error while loading shared libraries: libnss3.so: cannot open shared object file: No such file or directory\n\n\nTROUBLESHOOTING: https://github.com/puppeteer/puppeteer/blob/main/docs/troubleshooting.md\n\n    at onClose (/home/eribaeri/streetmerchant/node_modules/puppeteer/lib/cjs/puppeteer/node/BrowserRunner.js:193:20)\n    at Interface.<anonymous> (/home/eribaeri/streetmerchant/node_modules/puppeteer/lib/cjs/puppeteer/node/BrowserRunner.js:183:68)\n    at Interface.emit (events.js:327:22)\n    at Interface.close (readline.js:424:8)\n    at Socket.onend (readline.js:202:10)\n    at Socket.emit (events.js:327:22)\n    at endReadableNT (_stream_readable.js:1327:12)\n    at processTicksAndRejections (internal/process/task_queues.js:80:21)"
}

Also i wasn't able to install ca-certificates with the command from Wiki

Please help me, i need the program to run on my laptop since i am not home too often. 
Greetings,
Erik

**Comment by AlphaOmega2020** [comment]
1. Clone this project git clone https://github.com/jef/streetmerchant.git
- To checkout a particular ref, use git checkout <ref name> after cloning
2. Run npm install
3. Copy .env-example to a new file .env and edit the .env file to your liking using your favorite text editor
- More on this in customization
4. Run npm run start to start

**Comment by AlphaOmega2020** [comment]
.env example: https://github.com/jef/streetmerchant/blob/main/.env-example

**Comment by Brusalk** [comment]
Related: 
https://github.com/jef/streetmerchant/issues/341 
https://github.com/jef/streetmerchant/wiki/Installation:-Linux:-Debian-based

I was missing the packages listed in the second link - once I apt-installed them chrome started working.
